docs/src/howto/install/charm.md

```diff
@@ -0,0 +1,87 @@
+# Install Canonical Kubernetes from a charm
+
+Canonical Kubernetes is packaged as a [charm], available from charmhub for all supported platforms.
+
+## What you'll need
+
+This guide assumes the following:
+
+- The rest of this page assumes you already have Juju installed and have added [credentials] for a cloud and bootstrapped a controller.
+- If you still need to do this, please take a look at the quickstart instructions, or, for custom clouds (OpenStack, MAAS), please consult the [Juju documentation][juju].
```

Comment:
Could we add the quickstart instructions link here please?

Reply:
The quickstart isn't really relevant to the charm deployment, but I will add some more explanation